### PR TITLE
fix wss:// support

### PR DESCRIPTION
**Pull request by aqiank**
Signed-off-by: Jacky Boen <jacky@ciirobot.com>

---
This change is [Reviewable](https://reviewable.io/reviews/robotwebtools/roslibjs/233)

### CODE REVIEW

File: build/roslib.js [hunk 1]
```diff
@@ -12,7 +12,51 @@ function toObject(val) {
 	return Object(val);
 }
 
-module.exports = Object.assign || function (target, source) {
+function shouldUseNative() {
```
Comment (jihoonl) [inline build/roslib.js]: Could you delete test code?

**Comment by aqiank** [comment]
I deleted the test code in build/roslib.js. Is that okay?

**Comment by jihoonl** [comment]
@aqiank Sorry for late response. Could you remove all changes in the build directroy? roslib.js and roslib.min.js need to be clean.

**Comment by jihoonl** [comment]
THis PR will fix https://github.com/RobotWebTools/roslibjs/issues/228

**Comment by aqiank** [comment]
Done

**Comment by jihoonl** [comment]
:+1: thanks.

**Pull request merged**